bazel/grpc_python_deps.bzl

```diff
@@ -1,8 +1,67 @@
-load("//third_party/py:python_configure.bzl", "python_configure")
-load("@io_bazel_rules_python//python:pip.bzl", "pip_repositories")
-load("@grpc_python_dependencies//:requirements.bzl", "pip_install")
+"""Load dependencies needed to compile and test the grpc python library as a 3rd-party consumer."""
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@com_github_grpc_grpc//third_party/py:python_configure.bzl", "python_configure")
 
 def grpc_python_deps():
+    native.bind(
+        name = "six",
+        actual = "@six_archive//:six",
+    )
+
+    # protobuf binds to the name "six", so we can't use it here.
```

Comment:
I'm afraid it's not quite that simple. Removing the bind would be a breaking change, both for us and for the `protobuf` repo.

Reply:
Curious: Does ProtoBuf using the name "six" for the Python package `six`? In that case, does our external packages prevent our dependents unable to reuse those names?

Reply:
The fact that `protobuf` uses `bind` on the name `six` means that no project that transitively depends on it can use the name `six`. I really wanted `six` to be referred to as `@six//:six`, but Bazel throws an esoteric error that does not at all indicate the problem is a name collision. That wall was a tough one to crack through. I don't see any option besides naming the actual repo something besides `six` and binding it to `//external:six`.